treescript/Dockerfile.test

```diff
@@ -7,7 +7,7 @@ WORKDIR /app
 COPY MANIFEST.in setup.py tox.ini /app/
 COPY requirements/ /app/requirements/
 
-RUN pip install -r requirements/local.txt
+RUN pip install -r requirements/local.py37.txt
```

Comment:
I believe releng is mostly on py38 now

Reply:
I don't recall why I had changed that. I've been working with py38.